lib/new_relic/agent/instrumentation/active_support_logger.rb

```diff
@@ -10,13 +10,7 @@
   named :active_support_logger
 
   depends_on do
-    defined?(ActiveSupport::Logger) &&
-      # TODO: Rails 7.1 - ActiveSupport::Logger#broadcast method removed
-      # APM logs-in-context automatic forwarding still works, but sends
-      # log events for each broadcasted logger, often causing duplicates
-      # Issue #2245
-      defined?(Rails::VERSION::STRING) &&
-      Gem::Version.new(Rails::VERSION::STRING) < Gem::Version.new('7.1.0')
+    defined?(ActiveSupport::Logger) && defined?(ActiveSupport::Logger.broadcast)
```

Comment:
`ActiveSupport::Logger` is still defined in Rails 7.1, but the method we instrument, `ActiveSupport::Logger.broadcast` is not. This seemed more precise than using a Rails version.